migrations/versions/0340_stub_training_broadcasts.py

```diff
@@ -0,0 +1,25 @@
+"""
+
+Revision ID: 0340_stub_training_broadcasts
+Revises: 0339_service_billing_details
+Create Date: 2021-01-26 16:48:44.921065
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0340_stub_training_broadcasts'
+down_revision = '0339_service_billing_details'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('broadcast_message', sa.Column('stubbed', sa.Boolean(), nullable=True))
```

Comment:
we should add a second PR to make this column non-nullable later (but we'll need a second PR so we can ensure new rows always get a value, and then in the new script we can migrate old rows and set the column to nullable=False)

Reply:
that sounds good 👍🏼